vars/loadResources.groovy

```diff
@@ -1,17 +1,24 @@
+import io.openshift.Utils
+
 def call(Map args = [:]) {
-  if(!args.file) {
+  if (!args.file) {
     error "Missing manadatory parameter: file"
   }
 
   def yaml = readYaml(file: args.file)
+  def validate = args.validate ?: "true"
```

Comment:
have you validated that this wouldn't set `validate` to `true` even if it is set to `false`?

Reply:
yeah issue is if `args.validate` is set to `false` then `args.validate ?: true` is always evaluates to `true`

better way make it as `def validate = args.validate == false ? false : true` if we want to use boolean type

Reply:
@sthaha Yes I test it. But now as we have changed variable to boolean, it works fine.
@hrishin thanks.